internal/resources/resource_lb_virtual_servers.go

```diff
@@ -42,60 +48,99 @@ func LoadBalancerVirtualServers() *schema.Resource {
 				Required:    true,
 				Description: "pool of Network loadbalancer virtual server",
 			},
-			"ssl_cert": {
-				Type:        schema.TypeInt,
+			"vip_host_name": {
+				Type:        schema.TypeString,
+				Optional:    true,
+				Description: "vip_host_name of Network loadbalancer virtual server",
+			},
+			"type": {
+				Type: schema.TypeString,
+				ValidateDiagFunc: validations.StringInSlice([]string{
+					"http",
+					"tcp",
+					"udp",
+				}, false),
+				Required:     true,
+				InputDefault: "http",
+				Description:  "vip protocol of Network loadbalancer virtual server",
+			},
+			"tcp_application_profile":  schemas.TCPAppProfileSchema(),
+			"udp_application_profile":  schemas.UDPAppProfileSchema(),
+			"http_application_profile": schemas.HTTPAppProfileSchema(),
+			"persistence": {
+				Type: schema.TypeString,
+				ValidateDiagFunc: validations.StringInSlice([]string{
+					"SOURCE_IP",
+					"COOKIE",
```

Comment:
Can it not have DISABLED ?

Reply:
in UI it is showing as DISABLED but in json for persistence is empty
Please refer the below screenshot
<img width="500" alt="ERR" src="https://user-images.githubusercontent.com/61727100/182876544-7f96193b-82af-43fc-9bb1-3f63c2cfea4f.PNG">